source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetCompactStyleView.h

```diff
@@ -36,7 +36,13 @@
 
 - (NSString *)getItemAt:(NSInteger)index;
 
-- (void)filter:(NSString *)filter;
+- (BOOL)findMatch:(NSString *)queryString;
+
+- (NSRange)getHighlightRangeForSearchText:(NSString *)searchText resultText:(NSString *)resultText;
+
+- (void)updatefilteredListForStaticTypeahead:(NSString *) choices;
```

Comment:
nit: updateFilter.. instead of updatefilter..

Reply:
updated